operators/src/mock/mock_feature_collection_source.rs

```diff
@@ -141,7 +141,7 @@ mod tests {
 
         let serialized = serde_json::to_string(&source).unwrap();
         let collection_bytes = [
-            65, 82, 82, 79, 87, 49, 0, 0, 144, 1, 0, 0, 16, 0, 0, 0, 0, 0, 10, 0, 14, 0, 12, 0, 11,
+            65, 82, 82, 79, 87, 49, 0, 0, 148, 1, 0, 0, 16, 0, 0, 0, 0, 0, 10, 0, 14, 0, 12, 0, 11,
```

Comment:
is there a new test dataset?

Reply:
No, but this is the flight format. I was also curious why this changed, but the byte representation is very suboptimal for now. In the future, we should use the json writer if it is available.